docs/setup_guides/hpc.rst

```diff
@@ -348,3 +448,9 @@ And the second one a few times
 You can run this more times during your session to increase your
 allocation dynamically. You can also kill these jobs independently to
 contract your allocation dynamically and save compute time.
+
+.. _conda: https://conda.io/docs/
+.. _Jupyter: https://jupyter.org/
+.. _Dask: https://dask.pydata.org/
+.. _Cheyenne: https://www2.cisl.ucar.edu/resources/computational-systems/cheyenne
+.. _dask-jobqueue: http://dask-jobqueue.readthedocs.io
```

Comment:
It would be good to point people to [Dask's HPC setup docs](http://dask.pydata.org/en/latest/setup/hpc.html) at some point.

Reply:
I added a section at the bottom for further reading.